### PR TITLE
extracting annotations from input notebooks - add ontology to the graph - default values from notebooks parameters

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
Thanks to https://github.com/oda-hub/nb2workflow/issues/69 values for the parameters extracted from the two notebooks are now displayed

image

File: setup.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
     'renku==2.2.0',
     'astroquery @ git+https://github.com/oda-hub/astroquery#egg=astroquery',
     'aqsconverters @ git+https://github.com/oda-hub/aqsmodel-converters@simple-autolog#egg=aqsconverters',
-    'nb2workflow @ git+https://github.com/oda-hub/nb2workflow.git#egg=nb2workflow',
+    'nb2workflow @ git+https://github.com/oda-hub/nb2workflow@fix-jinja2-version#egg=nb2workflow',
```
Comment (volodymyrss) [inline setup.py:29]: this will be fixed soon presumably, so let's wait for that

File: setup.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
     'renku==2.2.0',
     'astroquery @ git+https://github.com/oda-hub/astroquery#egg=astroquery',
     'aqsconverters @ git+https://github.com/oda-hub/aqsmodel-converters@simple-autolog#egg=aqsconverters',
-    'nb2workflow @ git+https://github.com/oda-hub/nb2workflow@fix-jinja2-version#egg=nb2workflow',
+    'nb2workflow',
```
Comment (volodymyrss) [inline setup.py:29]: it's better to set `>=...` once it's bumped

Comment (burnout87) [reply setup.py]: Was it umpoed after https://github.com/oda-hub/nb2workflow/pull/92 ?

**Review by volodymyrss — APPROVED** [review]
I think we can think of refactoring when taking out non-astro part

File: setup.py (hunk 2 above)
Comment (burnout87) [reply setup.py]: I upgraded the version but will it bump automatically?

Comment (volodymyrss) [reply setup.py]: I am not sure we understand word "bump" in the same way here. Bump means to me increment version. Anyway you seem to have added the constrain so it's enough.

Comment (burnout87) [reply setup.py]: yes that is what I meant, VERSION increment, thanks

**Pull request merged**